src/layouts/default.vue

```diff
@@ -1,5 +1,23 @@
 <template>
   <div class="flex flex-col items-stretch min-h-screen">
+    <AlertBanner
+      :primary-button-text="$t('alert_banner_actions_purchase')"
```

Comment:
Do we need v-if `$t('alert_banner_content_heavenly_creations_life_like')` or sth?

Reply:
Probably remove the whole component instead of removing the i18n key if not used